__tests__/app.test.js

```diff
@@ -12,23 +13,35 @@ afterAll(() => {
   return db.end();
 });
 
+describe("/api", () => {
+  describe("GET: status: 200", () => {
+    test("responds with an array of available endpoints", () => {
+      const expectedResponse = endpoints;
+      return request(app)
+        .get('/api')
+        .expect(200)
+        .then((response) => {
+          expect(response.body.endpoints).toEqual(expectedResponse);
+      })
+    });
+  });
+});
+
 describe("/api/topics", () => {
-  describe("GET:", () => {
-    describe("status 200:", () => {
-      test("responds with array of topics with slug and decription", () => {
-        return request(app)
-          .get("/api/topics")
-          .expect(200)
-          .then((result) => {
-            const topics = result.body.topics;
-            if (topics.length > 0) {
-              topics.forEach((topic) => {
-                expect(topic).toHaveProperty("description");
-                expect(topic).toHaveProperty("slug");
-              });
-            }
-          });
-      });
+  describe("GET: status 200:", () => {
+    test("responds with array of topics with slug and decription", () => {
+      return request(app)
+        .get("/api/topics")
+        .expect(200)
+        .then((response) => {
+          const topics = response.body.topics;
+          if (topics.length > 0) {
```

Comment:
We should check whether we are getting the correct amount of topics back, not just any number above 0!